static/app/utils/tokenizeSearch.tsx

```diff
@@ -33,10 +33,22 @@ function isParen(token: Token, character: '(' | ')') {
 // searchSyntax/parser. We should absolutely replace the internals of this API
 // with `parseSearch`.
 
-export class QueryResults {
+export class MutableSearch {
   tokens: Token[];
 
-  constructor(strTokens: string[]) {
+  /**
+   * Creates a MutableSearch from a string query
```

Comment:
```suggestion
   * Creates a mutable search from a string query
```

Reply:
II think it's OK for this to give the type name